File: resources/expected/ros2/python-isolette_inverted_topics/strict/src/isolette_py_pkg/isolette_py_pkg/base_code/thermostat_regulate_temperature_manage_regulator_interface_mrit_runner.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from rclpy.executors import MultiThreadedExecutor
from isolette_py_pkg.user_code.thermostat_regulate_temperature_manage_regulator_interface_mrit_src import thermostat_regulate_temperature_manage_regulator_interface_mrit
#========================================================
# Re-running Codegen will overwrite changes to this file
#========================================================
def main(args=None):
    rclpy.init(args=args)
    node = thermostat_regulate_temperature_manage_regulator_interface_mrit()
    executor = MultiThreadedExecutor()
    executor.add_node(node)
    executor.spin()
    rclpy.shutdown()

if __name__ == "__main__":
    main()

File: resources/expected/ros2/isolette/strict/src/isolette_cpp_pkg/src/base_code/thermostat_regulate_temperature_manage_regulator_interface_mrit_runner.cpp
#include "isolette_cpp_pkg/user_headers/thermostat_regulate_temperature_manage_regulator_interface_mrit_src.hpp"

//=================================================
//  D O   N O T   E D I T   T H I S   F I L E
//=================================================

thermostat_regulate_temperature_manage_regulator_interface_mrit::thermostat_regulate_temperature_manage_regulator_interface_mrit() : thermostat_regulate_temperature_manage_regulator_interface_mrit_base()
{
    // Invoke initialize entry point
    initialize();

    PRINT_INFO("thermostat_regulate_temperature_manage_regulator_interface_mrit infrastructure set up");
}

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto executor = rclcpp::executors::MultiThreadedExecutor();
    auto node = std::make_shared<thermostat_regulate_temperature_manage_regulator_interface_mrit>();
    executor.add_node(node);
    executor.spin();
    rclcpp::shutdown();
    return 0;
}